src/go/configgenerator/route_generator_test.go

```diff
@@ -71,121 +71,123 @@ func TestMakeRouteConfig(t *testing.T) {
 			},
 			wantRouteConfig: `
 {
-    "name": "local_route",
-    "virtualHosts": [
+  "name": "local_route",
```

Comment:
nit: did you change indentation? diff is big

Reply:
Yeah, I used go/jsonformatter to pretty print the json which uses two space for indent I think.